File: app/src/main/java/com/wealoha/social/api/user/MatchSettingData.java
package com.wealoha.social.api.user;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

import com.wealoha.social.beans.ResultData;
import com.wealoha.social.beans.region.Regions;

public class MatchSettingData extends ResultData implements Serializable {

	private static final long serialVersionUID = -6856543043928783808L;
	public static final String TAG = MatchSettingData.class.getName();
	public boolean filterEnable;
	public String filterRegion;
	public Integer filterAgeRangeStart;
	public Integer filterAgeRangeEnd;
	public ArrayList<Regions> regions;
	public List<String> selectedRegion;

}

File: app/src/main/java/com/wealoha/social/beans/MatchSettingData.kt
package com.wealoha.social.beans

import java.io.Serializable
import java.util.*

class MatchSettingData : ResultData(), Serializable {
    @JvmField
    var filterEnable = false
    @JvmField
    var filterRegion: String? = null
    @JvmField
    var filterAgeRangeStart: Int? = null
    @JvmField
    var filterAgeRangeEnd: Int? = null
    @JvmField
    var regions: ArrayList<Regions>? = null
    @JvmField
    var selectedRegion: List<String>? = null

    companion object {
        private const val serialVersionUID = -6856543043928783808L
        @JvmField
        val TAG = MatchSettingData::class.java.name
    }
}